### PR TITLE
Checks if navLogo is a dotAsset

### PR DESCRIPTION
**Pull request by ivorpad**
If is not then we display the default logo

### CODE REVIEW

File: apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export class DotNavLogoService {
      * @memberof DotNavLogoService
      */
     setLogo(navLogo: string): void {
-        if (!navLogo) {
+        if (!navLogo || !navLogo.startsWith('/dA')) {
```
Comment (alfredo-dotcms) [inline apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.ts:19]: test?

Comment (fmontes) [reply apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.ts:19]: Yup this need test

**Review by fmontes — CHANGES_REQUESTED** [review]

**Review by alfredo-dotcms — DISMISSED** [review]

File: apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.spec.ts [hunk 2]
```diff
@@ -28,4 +28,12 @@ describe('DotNavLogoService', () => {
             done();
         });
     });
+
+    it("should not set a logo if the logo string doesn't starts with /dA", (done) => {
```
Comment (alfredo-dotcms) [inline apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.spec.ts:32]: I would add another test with the prefix `/dA`

Comment (ivorpad) [reply apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.spec.ts:32]: @alfredo-dotcms Wouldn't it be redundant with this test? https://github.com/dotCMS/core-web/blob/e479330a878426f6b089f18aeb75c328ff3f277f/apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.spec.ts#L24-L30

File: apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.ts (hunk 1 above)
Comment (ivorpad) [reply apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.ts:19]: Done

File: apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.spec.ts (hunk 2 above)
Comment (alfredo-dotcms) [reply apps/dotcms-ui/src/app/api/services/dot-nav-logo/dot-nav-logo.service.spec.ts:32]: you're right it's already covered!

**Pull request merged**